README.md

```diff
@@ -106,6 +106,29 @@ projects.  For this feature to work, you need to obtain a Google API key, and st
 it in `web/clientId.txt`.  If you don't do this, the sign-in and save features will
 not function correctly, but the rest of the site will be usable.
 
+To get a Google API key for your local CodeWorld installation, running at
+`http://localhost:8080`, follow these step:
+
+1. Go to [Google API Console](https://console.cloud.google.com/apis/)
```

Comment:
I wonder if it would be better to link to existing Google documentation for some of these steps.  Here's what I found:

* [Creating a project](https://cloud.google.com/resource-manager/docs/creating-managing-projects)
* [Creating an OAuth client id](https://support.google.com/cloud/answer/6158849?hl=en)